src/tensors/backends.jl

```diff
@@ -0,0 +1,32 @@
+# Scheduler implementation
+# ------------------------
+function select_scheduler(scheduler=OhMyThreads.Implementation.NotGiven(); kwargs...)
+    return if scheduler == OhMyThreads.Implementation.NotGiven() && isempty(kwargs)
+        Threads.nthreads() > 1 ? SerialScheduler() : DynamicScheduler()
```

Comment:
Is this the right order? `nthreads() > 1`, then use `SerialScheduler()`. So if `nthreads()==1`, then use `DynamicScheduler()`?